src/App4.js

```diff
@@ -0,0 +1,31 @@
+React.useEffect(() => {
+    if(age && name == '1') {
+      setName('2')
+    }
+    console.log('hello');
+    if(true) {
+      setAge(10)
+    } else if(age === 34) {
+      setAge(36)
+    }
+  }, [])
+
+function App2() {
+  const [name, setName] = React.useState('asd');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/shekhar-rudder/code-code/pullRequest?prid=7912542)
```suggestion
  const [name, setName] = React.useState("asd");
```